src/main/java/uk/gov/hmcts/darts/common/repository/AutomatedTaskRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.darts.common.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.darts.common.entity.AutomatedTaskEntity;
+
+import java.util.List;
+
+@Repository
+public interface AutomatedTaskRepository extends JpaRepository<AutomatedTaskEntity, Integer> {
+
+    AutomatedTaskEntity findByTaskName(String taskName);
```

Comment:
Please change return type to `Optional<AutomatedTaskEntity>` per https://tools.hmcts.net/confluence/display/DMP/Best+Practices#BestPractices-RepositoryMethodsShouldReturnOptional%3C%3EForAllNon-CollectionReturnTypes.